app/views/spree/components/products/_product_taxons.html.erb

```diff
@@ -1,13 +1,13 @@
 <% if !@product.taxons.blank? %>
   <section class="product-taxons">
     <h2 class="product-taxons__title">
-      <%= t('spree.look_for_similar_items') %>
+      <%= t('spree.look_for_similar_items') %>:
     </h2>
 
     <ul class="product-taxons__list">
       <% @product.taxons.each do |taxon| %>
         <li>
-          <%= link_to taxon.name, seo_url(taxon) %>
+          <%= link_to taxon.name, seo_url(taxon), class: "link-accent-color" %>
```

Comment:
I'd avoid these `link-accent-color` classes in favor of adding this style in the CSS. If we want to keep a class in the HTML it should have a semantic meaning. Why does this link have a different style? If it's an appearance concern, HTML should not be aware of that.